rhscl-charter.md

```diff
@@ -343,7 +343,7 @@ set | grep -e '^MYSQL_CONFIG_'|sed -e 's/^MYSQL_CONFIG_//' >>/etc/my.cnf.d/gener
 
 ### httpd dockerfile:
 
-* `EXPOSE` 80, 443
+* `EXPOSE` 8080, 8443
 * Config dir: `/etc/httpd`
 * Daemon runs as `apache` (USER directive)
 * Log file: `/var/log/httpd/`
```

Comment:
Everything should be logged into standard output so it can be captured by Docker and then forwarded to users via `docker logs container_id` or later into journald using the journald plugin. Having logs on container or volume does not make sense and it makes the container grow in size.